packages/material-ui/src/Accounts.js

```diff
@@ -1,28 +1,33 @@
 import React, { PropTypes } from 'react';
 import { Accounts as AccountsBase } from '@accounts/react';
-import { merge } from 'lodash';
+import merge from 'lodash/merge';
 import _loginComponents from './loginComponents';
 import _signupComponents from './signupComponents';
 import _forgotPasswordComponents from './forgotPasswordComponents';
+import _resetPasswordComponents from './resetPasswordComponents';
 
-const Accounts = ({ loginComponents, signupComponents, forgotPasswordComponents, ...otherProps }) =>
-  <AccountsBase
-    loginComponents={merge({}, _loginComponents, loginComponents)}
-    signupComponents={merge({}, _signupComponents, signupComponents)}
-    forgotPasswordComponents={merge({}, _forgotPasswordComponents, forgotPasswordComponents)}
-    {...otherProps}
-  />;
+const Accounts = ({ loginComponents, signupComponents, forgotPasswordComponents,
```

Comment:
Just a thought: what about adding a `commonComponents` prop which also is included in the merge. For example if you want to have a custom `Avatar` component which is the same between all the forms it needs to be passed in 4 times, once for every prop.